src/status_im/ui/screens/group/events.cljs

```diff
@@ -1,6 +1,24 @@
 (ns status-im.ui.screens.group.events
-  (:require [status-im.utils.handlers :as handlers]
-            [status-im.ui.screens.group.navigation]))
+  (:require [cljs.reader :as reader]
+            [re-frame.core :as re-frame]
+            [status-im.utils.js-resources :as js-res]
+            [status-im.utils.handlers :as handlers]
+            [status-im.ui.screens.group.navigation])
+  (:require-macros [status-im.utils.slurp :refer [slurp]]))
+
+;;;; COFX
+
+(re-frame/reg-cofx
+ :get-default-contacts
+ (fn [coeffects _]
+   (assoc coeffects
+          :default-contacts js-res/default-contacts)))
+
+(re-frame/reg-cofx
+ :get-default-dapps
+ (fn [coeffects _]
+   (assoc coeffects
+          :default-dapps (reader/read-string (slurp "./src/status_im/default-dapps.edn")))))
```

Comment:
what the reason to have edn instead clj?

Reply:
Will change to clj